src/ring_theory/localization.lean

```diff
@@ -743,6 +744,89 @@ end away
 
 end away
 
+section inv_submonoid
+
+variables (M S) [is_localization M S]
+
+lemma image_invertible : M.map (algebra_map R S : R →* S) ≤ is_unit.submonoid S :=
```

Comment:
I'm not completely satisfied with this suggestion, but maybe this name?
```suggestion
lemma map_le_is_unit_submonoid : M.map (algebra_map R S : R →* S) ≤ is_unit.submonoid S :=
```

Reply:
I'd prefer `submonoid_map_le_is_unit` more.